pvlib/modelchain.py

```diff
@@ -582,17 +582,28 @@ def with_pvwatts(cls, system, location,
             constructor and take precedence over the default
             configuration.
 
+        Warning
+        -------
+        The PVWatts model defaults to 14 % total system losses. The loss
+        assumptions can be modified as shown in the example below.
```

Comment:
```suggestion
        The PVWatts model defaults to 14 % total system losses. The loss
        assumptions represent annual averages and can be modified as
        shown in the example below.
```

Reply:
Two thoughts, neither of which I feel too strongly about:
- I think "annual averages" is true but not as specific as it could be.  How about "...represent constant DC-side loss fractions and can be modified..."?  As an aside, constant losses means all simulation timesteps are affected equally (on a relative basis), so the "snow loss" applies to July just as much as it does to December...
- Re "not recommended": I certainly don't disagree with the sentiment, but this struck me as too subjective a statement to be appropriate for pvlib's documentation.  If we want to make novice users aware that PVWatts v5 isn't a very detailed model, I think it would be better to describe what the model does (or doesn't) do, and let the user judge suitability for themselves.

Reply:
Bikeshedding: "The PVWatts losses are fractions of DC power and can be modified..."